tests/test_config.py.bad

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
That would cause a error during unit tests since it is not in right format as python codes. Check this [link](https://ci.appveyor.com/project/rhiever/tpot/build/1.0.79/job/wuosvq3apiysux7a)